eahub/localgroups/views.py

```diff
@@ -1,14 +1,38 @@
 import json
-from datetime import datetime 
+from datetime import datetime
 
 from django.urls import reverse_lazy
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-
+from django.core.mail import mail_managers
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib import messages
+from django.template.loader import render_to_string
 from .models import LocalGroup as Group
 
 def view_group(request, slug):
     group = get_object_or_404(Group, slug=slug)
+
     return render(request, 'eahub/group.html', {
         'group': group
     })
+
+@login_required(login_url=reverse_lazy('login'))
+def claim_group(request, slug):
+    group = get_object_or_404(Group, slug=slug)
+    if request.method == 'POST':
+        subject = "EA Group claimed: {0}".format(group.name)
+        user_eahub_url = "https://{0}/profile/{1}".format(get_current_site(request).domain,request.user.profile.slug)
+        message = render_to_string('emails/claim_group.html', {
+            'user_eahub_url': user_eahub_url,
+            'user_name': request.user.profile.name,
+            'group_name': group.name,
+            'user_email': request.user.email
+        })
+        mail_managers(subject, message)
```

Comment:
You can't pass HTML here. There are two options:

- Create two templates, one for HTML email and one for plain text, and use the `message` and `html_message` parameters.
- Create only the plain-text template and just use the `message` parameter.

The second option should suffice, I think, since this will only be sent to site staff.

Also, did you actually test this locally? Or did it not have HTML tags visible in the output when you did?

Reply:
It was working locally without HTML tags. Was following [this example](https://stackoverflow.com/a/28476681). So I think it should be fine to leave this as it is?